### PR TITLE
Profile/aw/hotfix eslint upgrade errors

### PR DESCRIPTION
**Pull request by adamwhitlock1**
## Summary

- upgrading from babel-eslint to @babel/eslint-parser caused some errors to be reported in the CI/CD. The original PR that  updated vats-website to use the new package has been reverted to prevent issues with daily deployment, so this PR aims to fix the linting issue so that upgrade can move forward. 
- fixes mostly invalid whitespace characters and some other minor eslint issues. There should be no actual changes to any site functionality.
- you can see the failing step here https://github.com/department-of-veterans-affairs/vets-website/actions/runs/5092299233/jobs/9153466905 and see the various linting errors at the top of the log

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/59030

## Testing done

- tested locally especially with whitespace to make sure text didn't appear incorrectly

## Screenshots

No visible UI changes

## What areas of the site does it impact?

no direct impact other than code style / linting based changes

### CODE REVIEW

File: src/applications/static-pages/facilities/vet-center/components/VetCenterImageSection.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
```
Comment (adamwhitlock1) [inline src/applications/static-pages/facilities/vet-center/components/VetCenterImageSection.jsx:1]: the eslint error seemed to be a false positive in this file, so it was ignored

**Review by acrollet — APPROVED** [review]
`coronavirus-vaccination-expansion` change approved

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by Mottie — APPROVED** [review]

**Review by maxx1128 — APPROVED** [review]

**Review by aaronyoung-tw — APPROVED** [review]

**Review by aaronyoung-tw — APPROVED** [review]

**Comment by nfstern02** [comment]
This looks like it's the frontend.
@Kernessant, Napoleon (governmentcio ***@***.***> or
@Simon, Jarred C. (governmentcio ***@***.***>
Can one of you kindly assist here. Not sure why this came to me.

From: Antonette Caldwell ***@***.***>
Sent: Thursday, August 3, 2023 10:41 AM
To: department-of-veterans-affairs/vets-website ***@***.***>
Cc: Stern, Noah F. (GovCIO LLC) ***@***.***>; Review requested ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/vets-website] Profile/aw/hotfix eslint upgrade errors (PR #24369)


Hello!

Could we get an update on whether this can be merged? My team is working on a different Next.js project, at version Next.js 13, and we would like to use the existing eslint rules by the VA, but this PR is a blocker to where we can use the VA eslint rules.

-

**Review by GovNapoleon — APPROVED** [review]

**Comment by adamwhitlock1** [comment]
@acald-creator I just merged this, so now we should be able to move forward with your updates.

**Comment by acald-creator** [comment]
@adamwhitlock1 Thank you!

**Pull request merged**